test/test.py

```diff
@@ -106,6 +107,25 @@ def test_one_bundle(self):
 
     def _assert_bundle(self, uuid, version, manifest, metadata_files, age_range, has_specimens=True):
         bundle = Bundle(uuid, version, manifest, metadata_files)
+        expected_disease_values = {'b2216048-7eaa-45f4-8077-5a3fb4204953': {'subcutaneous melanoma'},
```

Comment:
The expected disease value should be passed to the function. Just like age_range.  If it is passed as None it shouldn't be asserted.

I'm starting to see a pattern in your work. You typically make things work "your way" ignoring how the code base currently solves a similar problem. I call that precedent. Generally speaking, you should try to stick with precedent when making changes to a code base. Here' you are introducing a new pattern that deviates from precedent. Current precedent is that all bundle specifics are mentioned in the list of tuples above, the one where you added your test bundle. Here you deviate by having this dict in the body of the function and looking up a bundle-specific expectation from that dict.

If being consistent with precedent is infeasible or unsustainable, you have to refactor the code base to create a new, more sustainable approach.

Reply:
Yes, I should have discussed with you first before commiting this. The main problem was with this change is that there are differing methods with solving similar problems. I thought about solving this problem using a similar approach to AgeRange. But I didn't want to set a precedent where if we wanted  to check if certain fields are correct, we have to add another element to the tuple and another arg in `_assert_bundle(...)`. Otherwise, we will end up with a confusingly long tuple and method. Also, I thought finding a better approach wouldn't be feasible now, since we're in crunch time. I won't commit anything incongruent with the current precedent without discussing it with you.

Reply:
You don't need to necessarily discuss alternatives to what you feel is confusing or convoluted, prior to committing a solution. But what you commit has to be a holistic solution. It has to make the project better, even if only you think it is better. The lookup you added here is making it worse because it adds inconsistency. You may not share my opinion that inconsistency is worse than a consistent but ugly solution. But the ugly solution was my directive. In a previous comment I had already asked that you emulate how AgeRange is handled and I conceded that it would be ugly. We can take the consistently ugly discussion offline.